perception_eval/perception_eval/common/dataset_utils.py

```diff
@@ -450,6 +451,7 @@ def _sample_to_frame_2d(
         frame_name=frame_name,
         objects=objects_,
         raw_data=raw_data,
+        scene_token=nusc_sample["scene_token"],
```

Comment:
If you want to have also frame token for particular frame, consider replace'ing it with scene_token=nusc_sample["token"], this should give you frame token, which differs from scene token added index of frame in the end of token, so it is easy to get also scene_token from frame_token